### PR TITLE
screen reader notifier now react component

### PR DESCRIPTION
**Pull request by wtfil**
Update interface of screen reader notifier to expose the `remove` method, which could be used in component lifecycle

### CODE REVIEW

File: src/components/screen-reader-notification/screen-reader-notification.st.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.root {
+	position: 'fixed';
```
Comment (AviVahl) [inline src/components/screen-reader-notification/screen-reader-notification.st.css]: no need for the quotes

File: src/components/screen-reader-notification/screen-reader-notification.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import {SBStateless} from 'stylable-react-component';
+import styles from './screen-reader-notification.st.css';
+
+export type Stylesheet = Stylesheet;
+export const ScreenReaderNotification = SBStateless(
+    ({children}: {children: React.ReactNode}) => {
+        return (
+            <p
+                role="alert"
+                children={children}
```
Comment (AviVahl) [inline src/components/screen-reader-notification/screen-reader-notification.tsx]: why pass it as prop and not a regular child element?

File: src/components/screen-reader-notification/screen-reader-notification.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import {Stylesheet} from 'stylable';
+import {SBStateless} from 'stylable-react-component';
+import styles from './screen-reader-notification.st.css';
+
+export type Stylesheet = Stylesheet;
```
Comment (AviVahl) [inline src/components/screen-reader-notification/screen-reader-notification.tsx]: why is this here? is this related to the type error you've had?

File: src/components/screen-reader-notification/screen-reader-notification.tsx (hunk 2 above)
Comment (wtfil) [reply src/components/screen-reader-notification/screen-reader-notification.tsx]: My personal preferences when it comes to components with "simple" children. A bit better to read all props when you use the short tag

File: src/components/screen-reader-notification/screen-reader-notification.st.css (hunk 1 above)
Comment (wtfil) [reply src/components/screen-reader-notification/screen-reader-notification.st.css]: `+`

**Comment by wtfil** [comment]
waiting for https://github.com/wixplosives/stylable/pull/116

**Comment by wtfil** [comment]
@AviVahl This is quite challenging to test this king of functionality. I personally don't know, how to do it. Still collecting knowledges in this area though
